### DISCUSSION TITLE
Filamemt 3.x multiple column validation and error message

### DISCUSSION THREAD

**Discussion by asifka** [filamentphp/filament#14989] [Help]
filament form validation, How can I show a validation error message for unique column validation (like event_id and reg_no) instead of encountering SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry? Using Filament 3.2 and Laravel 11.9

**Comment by leandrocfe** [comment]
maybe [custom rules](https://filamentphp.com/docs/3.x/forms/validation#custom-rules) ?
